src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState, ChangeEvent } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const caseInsensitiveMovies = movie.title.toLocaleLowerCase();
+    const caseInsensitiveDesc = movie.description.toLocaleLowerCase();
+    const caseInsensitiveQuery = query.toLocaleLowerCase();
```

Comment:
You need to remove unnecessary spaces in search query.
Your tests failed because of that